src/Common/tests/Performance/PerfUtils.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.IO;
+
+namespace System
+{
+    /// <summary>Static helper class for performance tests</summary>
+    public static class PerfUtils
+    {
+        private static Random rand = new Random(1234132);
```

Comment:
@ianhays, it will be shared amongst multiple tests,right?  That seems problematic, given that the performance of some of the tests will be dependent on the data that gets generated.  I suggest we merge this PR, but then we go back and change this support to be something that each perf test instantiates, creating its own instance with a fixed seed, so that test ordering won't impact performance in this way.